appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/MetricsService.java

```diff
@@ -126,6 +140,80 @@ public void init() {
         }
     }
 
+    @Override
+    public void collect(MonitoringDataCollector rootCollector) {
+        MonitoringDataCollector metricsCollector = rootCollector.in("metric");
+        if (metricsEnabled != null) {
+            metricsCollector.collect("enabled", metricsEnabled);
+        }
+        if (metricsSecure != null) {
```

Comment:
`getSecureMetrics()` is marked as deprecated, please use the `isSecurityEnabled()`:
`metricsCollector.collect("securityEnabled", isSecurityEnabled());`